app/src/main/res/values/strings.xml

```diff
@@ -63,111 +63,111 @@
+    <string name="txt_press_icon_below_speak_2">Press the icon below to stop recording.</string>
+    <string name="txt_sentence_recorded">Sentence recorded. Listen to the recording.</string>
+    <string name="txt_sentence_record_failed">Recording failed. Please try again.</string>
+    <string name="txt_recorded_correct_or_wrong">If you spoke correctly, send the recording.\nIf you misspoke, you can record the sentence again.</string>
+    <string name="txt_listening_stopped">Listen to the whole recording to make sure it\'s correct.</string>
+    <string name="txt_listening_again_recording">Press the icon below to stop listening.</string>
+    <string name="txt_recording_too_long">Recording is too long. Max duration is 10 seconds.</string>
+    <string name="txt_sending_recording">Sending the recording...</string>
     <string name="txt_recording_sent">Recording sent!</string>
-    <string name="txt_sending_recording_failed">Sending recording failed!</string>
-    <string name="txt_sending_recording_failed_and_skip">Sending recording failed! Try again to send it otherwise press {{*{{skip_button}}*}} button</string>
-    <string name="txt_press_icon_below_listen_1">Press the icon below to start the clip</string>
-    <string name="txt_press_icon_below_listen_2">Press the icon below to stop the clip</string>
-    <string name="txt_clip_correct_or_wrong">The clip is correct? Press the Thumb up\nThe clip is wrong? Press the Thumb down</string>
-    <string name="txt_clip_again">Press the icon below to start the clip again</string>
-    <string name="txt_clip_validated_yes">Clip accepted!</string>
-    <string name="txt_clip_validated_no">Clip rejected!</string>
-    <string name="txt_error_1_try_again_tap_skip">Error. Try again, so tap on {{*{{skip_button}}*}} button</string>
-    <string name="txt_error_2_sending_failed">Error during the sending of the recording. The audio wasn\'t sent to server</string>
-    <string name="txt_sending_validation">Sending the validation ...</string>
-    <string name="txt_error_3_sending_validation_failed">Error during the sending of the validation. Try again or press {{*{{skip_button}}*}} button</string>
-    <string name="txt_closing">Closing ...</string>
-    <string name="txt_error_4_clips_no_available">Error. There aren\'t any other clips to validate for this language.</string>
+    <string name="txt_sending_recording_failed">Failed to send recording!</string>
+    <string name="txt_sending_recording_failed_and_skip">Failed to send recording! You can either try again, or press the {{*{{skip_button}}*}} button.</string>
+    <string name="txt_press_icon_below_listen_1">Press the icon below to start the clip.</string>
+    <string name="txt_press_icon_below_listen_2">Press the icon below to stop the clip.</string>
+    <string name="txt_clip_correct_or_wrong">If the clip is correct, press the thumbs up.\nIf the clip is wrong, press the thumbs down.</string>
```

Comment:
Why "thumbs" instead of "thumb"?

Reply:
Because in English we say thumbs up rather than thumb up. That's just the word. See here: https://emojipedia.org/thumbs-up/

Reply:
Uh, I didn't know! Thanks! 😄